content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -2,78 +2,382 @@
+   Duplicate` within the tax menu to create a copy of an existing similar tax. Then, select another
+   :guilabel:`Exoneration`, and :guilabel:`Save`. Repeat this process as many times as you need
+   different :guilabel:`Exoneration` nature taxes.
+
+.. tip::
+   **Rename** your taxes in the :guilabel:`Name` field according to their :guilabel:`Exoneration` to
+   differentiate them easily.
+
+.. image:: italy/italy-tax.png
    :align: center
-   :alt: Asynchronous sending on a customer invoice
+   :alt: External reverse charge settings
+
+On your invoice, select the corresponding tax you need in the :guilabel:`Taxes` field. You can find
+and edit additional info by opening the **XML** file of the issued invoice:
+
+- :guilabel:`SdI address (Codice Destinatario)`: must be filled for both **EU** or
+  **non-EU**;
+- :guilabel:`Country Id`: must contain the country of the foreign seller in the two-letter ISO
+  (Alpha-2) code (ex., `IT` for 'Italy');
+- :guilabel:`CAP`: must be filled with `00000`;
+- :guilabel:`Partita Iva` (**VAT number**): must contain the **VAT** number for **EU businesses**
+  and `OO99999999999` (double 'O' letter, not 'zero') for **non-EU businesses**. In case of private
+  customers without **VAT** number, use `0000000`;
+- :guilabel:`Fiscal Code`: for foreign entities without an actual **Codice Fiscale**, any
+  recognizable identifier is valid.
+
+Vendor bills
+~~~~~~~~~~~~
+
+Italian companies buying goods or services from EU countries or non-EU countries must send the
+information contained within the bill received to the **Agenzia delle Entrate**. Set the seller as
+:guilabel:`Cedente/Prestatore`, the buyer as :guilabel:`Cessionario/Committente`, and fill out the
+tax information to be paid to the **Agenzia delle Entrate**.
+
+The Italian buyer sends the **tax agency** via the :abbr:`SdI (Systema di Interscambio)` the
+information of the invoice received, setting the seller as :guilabel:`Cedente/Prestatore`, the buyer
+as the :guilabel:`Cessionario/Committente`, and filling out the missing tax information that must be
+paid to the **Agenzia delle Entrate**. Self-billing invoices or VAT invoice integrations must be
+issued and sent to the tax agency.
+
+**Three** types of configurations are technically identified by a code called :guilabel:`Tipo
+Documento` (document type):
+
+#. `TD17` - Buying **services** from **EU** and **non-EU** countries:
+
+   The foreign *seller* invoices a service with a **VAT-excluded** price, as it is not taxable in
+   Italy. The VAT is paid by the *buyer* in Italy;
+
+   - Within EU: the *buyer* records the invoice received in their VAT return due in Italy;
+   - Non-EU: the *buyer* sends themselves an invoice (i.e., **self-billing**).
+
+     Odoo exports a transaction as `TD17` if the following conditions are met:
+
+   - Is a vendor bill;
+   - At least one tax on the invoice lines targets the tax grids `VJ`;
```

Comment:
As noted below, detail that it's `VJ` because it's the part of the tax report that is reverse charge related